src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -243,36 +245,37 @@ Concept
         <div className="concept-form-wrapper">
           <div className="row form-container">
             <div className="col-lg-12 col-md-10 col-sm-12">
-              { existingConcept
-              && (
-              <CreateConceptForm
-                handleNewName={this.handleNewName}
-                nameRows={this.props.newName}
-                removeRow={this.removeNewName}
-                description={this.props.description}
-                editable={this.state.notEditable}
-                addDescription={this.addNewDescription}
-                removeDescription={this.removeDescription}
-                toggleUUID={this.handleUUID}
-                concept={concept}
-                path={path}
-                state={this.state}
-                handleChange={this.handleChange}
-                handleSelections={this.handleNameLocale}
-                handleDescription={this.handleDescriptionLocale}
-                handleSubmit={this.handleSubmit}
-                addDataFromRow={this.addDataFromRow}
-                addDataFromDescription={this.addDataFromDescription}
-                removeDataFromRow={this.removeDataFromRow}
-                pathName={this.props.match.params}
-                existingConcept={existingConcept}
-                isEditConcept={this.state.isEditConcept}
-                answer={this.props.answer}
-                addDataFromAnswer={this.addDataFromAnswer}
-                addAnswer={this.addNewAnswer}
-                removeAnswer={this.removeAnswer}
-              />
-              )
+              {existingConcept
+                && (
+                  <CreateConceptForm
+                    handleNewName={this.handleNewName}
+                    nameRows={this.props.newName}
```

Comment:
@ozezzy, I think it is best to destructure all props;

Reply:
this is showing as change due to linter formatting. I have removed all linter changes.